2018-09-14/solutions/khromets.php

```diff
@@ -0,0 +1,20 @@
+<?php
+$arr1 = [1,2,3,4,5];
+$arr2 = [2,4,8];
```

Comment:
why do you put your test data in the very beginning of the file, if you use it only below for your test?